testnets/atomonetestnet1/chain.json

```diff
@@ -0,0 +1,170 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "atomonetestnet1",
+  "chain_type": "cosmos",
+  "chain_id": "atomone-testnet-1",
+  "website": "https://atom.one",
```

Comment:
What about using https://github.com/atomone-hub/networks/tree/master/atomone-testnet-1 instead ?
The informations of https://atom.one are mainnet oriented, while the link above is only about the testnet.

Reply:
I had to revert, doesn't pass the CI checks